### PR TITLE
OCPBUGS-5113: Date&Time values are not showing as per browser default language

### PR DESCRIPTION
**Pull request by cyril-ui-developer**
### Before:
 Screenshot 2023-01-31 at 10 26 58 AM

### After
Screenshot 2023-01-31 at 10 27 12 AM

### CODE REVIEW

File: frontend/packages/console-app/src/components/user-preferences/language/getLastLanguage.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
 import { LAST_LANGUAGE_LOCAL_STORAGE_KEY } from './const';
 
-export const getLastLanguage = (): string => localStorage.getItem(LAST_LANGUAGE_LOCAL_STORAGE_KEY);
+export const getLastLanguage = (): string =>
+  localStorage.getItem(LAST_LANGUAGE_LOCAL_STORAGE_KEY) || navigator.language;
```
Comment (jhadvig) [inline frontend/packages/console-app/src/components/user-preferences/language/getLastLanguage.ts:4]: we should be checking the `languages` array
```suggestion
  localStorage.getItem(LAST_LANGUAGE_LOCAL_STORAGE_KEY) || navigator.languages[0] || navigator.language;
```

@rawagner could you elaborate?

Comment (cyril-ui-developer) [reply frontend/packages/console-app/src/components/user-preferences/language/getLastLanguage.ts:4]: > navigator.languages

@jhadvig What we need is the user browser preferred lng when the `localStorage` is cleared by the "Use the default browser language setting." action in user preference > lng tab. The `navigator.language` returns the lng while `navigator.languages` an array of lngs with index 0 equal to `navigator.language`. So I think it is better and clearer to use ``navigator.language` except if you know of any reason not to. Both will always return the same value.

**Review by spadgett — COMMENTED** [review]
Let's add a test so that we don't regress again.

**Comment by cyril-ui-developer** [comment]
@jhadvig Updated. PTAL

**Review by jhadvig — APPROVED** [review]
/lgtm

**Review by jhadvig — APPROVED** [review]
/lgtm
/retest

**Comment by jhadvig** [comment]
/retest

```
    DeploymentConfig
      61) creates the resource instance
      62) displays detail view for newly created resource instance
      63) displays a list view for the resource
      64) search view displays created resource instance
      65) edits the resource instance
{"component":"entrypoint","file":"k8s.io/test-infra/prow/entrypoint/run.go:169","func":"k8s.io/test-infra/prow/entrypoint.Options.ExecuteProcess","level":"error","msg":"Process did not finish before 6h0m0s timeout","severity":"error","time":"2023-10-04T21:05:07Z"}
{"component":"entrypoint","file":"k8s.io/test-infra/prow/entrypoint/run.go:267","func":"k8s.io/test-infra/prow/entrypoint.gracefullyTerminate","level":"error","msg":"Process did not exit before 15s grace period","severity":"error","time":"2023-10-04T21:05:22Z"}
{"component":"entrypoint","error":"process timed out","file":"k8s.io/test-infra/prow/entrypoint/run.go:84","func":"k8s.io/test-infra/prow/entrypoint.Options.internalRun","level":"error","msg":"Error executing test process","severity":"error","time":"2023-10-04T21:05:22Z"}
error: failed to execute wrapped command: exit status 127 
INFO[2023-10-04T21:05:23Z] Step e2e-gcp-console-test failed after 6h10m12s. 
INFO[2023-10-04T21:05:23Z] Step phase test failed after 6h10m12s.       
INFO[2023-10-04T21:05:23Z] Running multi-stage phase post           
```

**Comment by cyril-ui-developer** [comment]
Fixed the pf-5 warning  errors but the CSS styling got broken after rebased with the `pf-5` changes. Remaining CSS styling issues are: 
- The mobile kebab is aligned left. Should be right of the screen. The `align={{ default: 'alignRight' }` seems not to work for `ToolbarGroup` after rebased with the `pf-5` changes.
- The divider is not properly aligned to the external pod log link for desktop screen
- Clicking on the kebab action throws a warning error due to the wrapping of `Button` with  `Dropdown` component. Seems `pf` 'Dropdown` is made from HTML `button` element.  So button cannot wrap button. Using either `Button` or  `Dropdown` component would require addig `pf-5` style to fix. Will figure out how to handle this later. 

See attached:
Screenshot 2023-11-22 at 2 16 28 PM
Screenshot 2023-11-22 at 2 20 02 PM

**Pull request merged**